CSS.css

```diff
@@ -33,18 +34,32 @@ h3 {
 }
 
 h4 {
-  color: #FCFBE3;
+  color: white;
 }
 
 
-ul {
-  color: #FCFBE3;
+
+<!-- Nav bar stuff -->
+
+
+
+ul.nav li a, ul.nav li a:visited {
+  color: orange !important;
+}
+
+<<<<<<< HEAD
+ul.nav li a:hover, ul.nav li a:active {
+  color: green !important;
```

Comment:
`!important` should never be used. If you need to override styles that cascade, use specific selectors or use **IDs** as these dominate over **classes**